provision.azure.yaml

```diff
@@ -0,0 +1,3 @@
+resources:
+  topics:
+    - name: comms
```

Comment:
This exact file also exists on the `fcp-fd-comms` service, just thinking for `fcp-fd-data` that `comms` should be replaced with `data` to match the service name?

Reply:
this was the pain of this PR. We were assuming the same, however after conversation with @johnwatson484 , we identified that we need to put 'comms' as value, if we want our service bus to correctly connect to the deployed pod PR pod

Reply:
Ah gotcha, makes sense 👍🏽.

Reply:
Yeah this was a confusing attaching Johns explanation....

<img width="442" alt="Screenshot 2024-11-29 at 13 50 24" src="https://github.com/user-attachments/assets/9b5cb2a8-7e0e-4553-850c-acfb4b2b8777">

because we've called our topic COMMS_TOPIC_ADDRESS we have to `comms` in the provision.azure.yaml` for it to all match up.

Reply:
Ahhh, gotcha! Cheers @simonnineteen90, stealing the screenshot to use as future reference 👀.